cluster_preflight_check/main.py

```diff
@@ -387,4 +387,7 @@ def run(context):
         sys.exit(1)
 
 
-ctx = Context()
+def main():
+    ctx = Context()
```

Comment:
should assign name here:
`ctx.name = os.path.basename(sys.argv[0])`
Thanks!

Reply:
Yes, that's a good idea. :+1:  I missed that part.

Reply:
hi @tomschr , could you please append that line? Then I can merge this PR:) Thanks

Reply:
@liangxin1300 Ahh, sorry for the delay. I missed your message.
I've included the line and pushed it into the branch so you can merge it. :wink: